buildkite-agent-scaler/variables.tf

```diff
@@ -0,0 +1,67 @@
+variable function_name {
+  type = string
+}
```

Comment:
We should add the descriptions from the buildkite parameters here. We can then use `terraform-docs` to create documentation from that.